end-to-end-tests/specs/specUtils.js

```diff
@@ -148,6 +148,10 @@ function checkOncoprintElement(selector) {
     return browser.checkElement(selector || "#oncoprintDiv", { hide:[".qtip", '.dropdown-menu', ".oncoprintjs__track_options__dropdown", ".oncoprintjs__cell_overlay_div"] });
 }
 
+function testJqueryExpression(callback){
```

Comment:
badly named function - it does something much more general

Reply:
hurt feelings.  but you're right.  in fact, this doesn't really add much of anything to browser.execute except call value (which is not even documented!!!!).   maybe it's worth it though just for value

Reply:
sorry!! I have a very terse github style.. I should be more sensitive.

Reply:
how about 
```
testJqueryExpression(expression:string) {
    return browser.execute(()=>$(expression).length).value;
}
```